run_local.py

```diff
@@ -18,8 +18,8 @@
 ]
 
 browsers = [
-    'Local_FF',
-    #  'Local_Chrome',
+    #  'Local_FF',
```

Comment:
I switched to Local_FF and it said the gecko driver is missing.  Can we add that to the base docker image.

Reply:
Oh fun I must have a pip package installed or something . I'll figure it out as it worked on my box :/